src/coreclr/src/inc/utilcode.h

```diff
@@ -775,11 +775,11 @@ class CCompRC
     CRITSEC_COOKIE m_csMap;
 
     LPCWSTR m_pResourceFile;
-#ifdef HOST_UNIX
+#ifdef TARGET_UNIX
```

Comment:
This is not DACized, so it should not be needed. 

Also, this type is designed to run on HOST not TARGET.

Reply:
Actually, it looks like all this is dead code, so it would be best to delete it.